pyproject.toml

```diff
@@ -9,7 +9,7 @@ python = "^3.9"
 Authlib = "^0.15.5"
 bcrypt = "^3.2.0"
 boto3 = "^1.26"
-cpr-data-access = {git = "https://github.com/climatepolicyradar/data-access.git", tag="v0.5.3", extras = ["vespa"]}
+cpr-data-access = {git = "https://github.com/climatepolicyradar/data-access.git", branch="feature/pdct-965-parameter-handling-needs-deduplicating", extras = ["vespa"]}
```

Comment:
Eeeek - can this be a tag please??

Reply:
Yep will do! I hadn't merged the other PR yet which is all it was waiting for

Reply:
Done!